openedx_webhooks/services/jira/utils.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+from __future__ import (
+    absolute_import, division, print_function, unicode_literals
+)
+
+import logging
+
+from . import jira as _jira
+from .models import JiraFields
+
+logger = logging.getLogger(__name__)
+
+
+def find_allowed_values(project_key, issue_type_name, field_name, jira=_jira):
+    meta = jira.createmeta(
+        project_key,
+        issuetypeNames=issue_type_name,
+        expand='projects.issuetypes.fields',
+    )
+    fields = meta['projects'][0]['issuetypes'][0]['fields']
+    field_id = make_fields_lookup([field_name])[field_name]
+    return fields[field_id]['allowedValues']
+
+
+def find_issues_for_pull_request(pull_request_url, jira=_jira):
+    jql = 'project=OSPR AND url="{}"'.format(pull_request_url)
+    return jira.search_issues(jql)
+
+
+def make_edx_action_choices():
```

Comment:
This function (and the constant in services/jira/constants.py) seems really tightly coupled to the update_latest_github_activity handler. They'd be easier to understand if you put this function into jira/tasks/update_latest_github_activity.py